wwwapp/admin.py

```diff
@@ -51,3 +52,5 @@ def make_clear(self, _request, queryset):
 admin.site.register(WorkshopType)
 admin.site.register(WorkshopParticipant)
 admin.site.register(WorkshopUserProfile)
+
```

Comment:
Why leave a vertical space here?

Reply:
Because other things are in groups like this, probably :P